docs/clients/index.md

```diff
@@ -136,7 +136,7 @@ operation, it can be automatically generated. See the [contribution guide](/cont
 for more information.
 
 | Api Client                                  | Package name                                                                                              |
-| ------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
+|---------------------------------------------| --------------------------------------------------------------------------------------------------------- |
```

Comment:
should be reverted. Or same change should be applied to the next column

Reply:
This is PHPStorm changing that when handling column formatting during the edition (no idea why it did it automatically only for the first column). I will change the other column